lms/views/basic_lti_launch.py

```diff
@@ -91,6 +93,23 @@ def store_lti_data(self):
                 request, h_user=lti_user.h_user, lti_user=lti_user
             )
 
+    @view_config(request_param="vitalsource_book=true", configured=False)
```

Comment:
This is necessary only if we need backwards compatibility with existing assignments.

As this is a "legacy view" that could be eventually removed I think having the `request_param` on the view_config is much more clear and independent that the existing predicate.

Reply:
Let's question whether we need that backwards compatibility with existing VitalSource assignments. I suspect we don't (it's not being used by real users yet) so the code can be simpler